src/ray/common/client_connection.cc

```diff
@@ -52,47 +52,16 @@ template <class T>
 Status ServerConnection<T>::WriteBuffer(
     const std::vector<boost::asio::const_buffer> &buffer) {
   boost::system::error_code error;
-  // Loop until all bytes are written while handling interrupts.
```

Comment:
Can we really get rid of this code? Same for below.

Reply:
From the [documentation](https://www.boost.org/doc/libs/1_37_0/doc/html/boost_asio/reference/write/overload1.html), it looks like it, but I guess I'm not completely sure.